SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,55 +1,70 @@
-{
-  GENERATED_BODY()
-  ...
-}
-```
+1. Open your C++ Class in any text editor.
+2. Tag the C++ Class with the relevant attribute(s), depending on what type of Singleton Actor you want it to be:
+   * **Public Singleton Actor**: tag it with the `SpatialType=Singleton` class attribute. The following code snippet shows how to tag a C++ Class as a Public Singleton Actor:
+     ```
+     UCLASS(SpatialType=Singleton)
+     class AScoreBoard : public AActor
+     {
+       GENERATED_BODY()
+       ...
+     }
+     ```
+   * **Private Singleton Actor**: tag it with the `SpatialType=(Singleton, ServerOnly)` class attributes. The following code snippet shows how to tag a C++ Class as a Private Singleton Actor:
+     ```
+     UCLASS(SpatialType=(Singleton, ServerOnly))
+     class AScoreBoard : public AActor
+     {
+       GENERATED_BODY()
+       ...
+     }
+     ``` 
 
-The code snippet below shows how to tag a native C++ class with the appropriate Private Singleton identifiers.
+### Unreal Blueprint Class
 
-```
-UCLASS(SpatialType=(Singleton, ServerOnly))
-class TESTSUITE_API AExampleGameGameMode : public AGameModeBase
-{
-  GENERATED_BODY()
-  ...
-}
-```
+1. Open your Blueprint Class in the [Blueprint Editor](https://docs.unrealengine.com/en-us/Engine/Blueprints/Editor).
+2. From the [Toolbar](https://docs.unrealengine.com/en-US/Engine/Blueprints/Editor/UIComponents/Toolbar), click **Class Settings** to open Blueprint Properties in the Default pane.
+3. In the **Class Options** section, ensure that the **Spatial Type** check box is selected.
+4. In the **Spatial Description** text box, tag the Blueprint Class with the relevant attribute(s), depending on what type of Singleton Actor you want it to be:
+   * **Public Singleton Actor**: enter `Singleton`.
+   * **Private Singleton Actor**: enter `Singleton ServerOnly`.
+  Here's a screenshot of what your Blueprint Class looks like if you tag it as a Private Singleton Actor:
 
-To tag a Blueprint class as a Public Singleton, open the class in the Blueprint Editor and navigate to the `Class Settings`. In the `Advanced` section inside `Class Options`, check the `Spatial Type` checkbox and add `Singleton` to the `Spatial Description` textbox. To tag a Blueprint class as a Private Singleton, follow the same steps and add `ServerOnly` to the `Spatial Description` textbox.
-
-This is an example of what your Blueprint `Class Options` should look like if you've tagged it as a Private Singleton:
 ![Singleton Blueprint]({{assetRoot}}assets/screen-grabs/blueprint-singleton.png)
 
-And that's it! You have successfully specified a Singleton Actor. Make sure that if you edit your Actor, you then generate [schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema) and create a new [snapshot]({{urlRoot}}/content/how-to-use-snapshots) using the [SpatialOS GDK toolbar]({{urlRoot}}/content/toolbars).
\ No newline at end of file
+## Managing Singleton Actors
```

Comment:
I think you've missed out the section on spawning?

Reply:
Thanks for pointing it out. I copied and pasted everything from the Gdocs here. When I tried to merge the changes, there were conflicts. It might occur while I was resolving the conflicts probably?

It's now added.